tenants-certified-attributes-updater/src/main/scala/it/pagopa/interop/tenantscertifiedattributesupdater/util/Utils.scala

```diff
@@ -96,7 +96,9 @@ object Utils {
         }
 
     val revocations: Map[PersistentExternalId, List[AttributeInfo]] =
-      fromTenant.toList
+      fromTenant
+        .filter(_._2.exists(_.revocationTimestamp.isEmpty))
```

Comment:
Can you name parameters? That's incredibly obscure.
Try something like

```scala
.filter {
 case (_, nameOfTheResource) => nameOfTheResource.exists(_.revocationTimestamp.isEmpty)
}
```

Reply:
Agree, plus: isn't this solving the problem just partially?

For example:
- the tuple is (Tenant1, List(Attribute1, Attribute2))
- Attribute1 is already revoked
- Attribute2 is not revoked

The logic would still revoke both Attribute1 and Attribute2, right?

Reply:
That will then require a list unpack with a flatmap or so

Reply:
The code filter the list of attributes when revocationTimestamp is defined.